### PR TITLE
Lit protocol tutorial & wagmi v2 updates

### PR DESCRIPTION
**Pull request by dtbuchholz**
## Summary

- Adds a walkthrough for how to use Lit for encrypting/decrypting data with your private key.
- Updates the wagmi/viem examples to use the new v2, which has breaking changes.
- Some small tweaks to clean up SQL FAQs and showcase pages.
- Small marketing updates to use the word "onchain" more.

### CODE REVIEW

File: docs/playbooks/frameworks/wagmi.md [hunk 1]
```diff
@@ -33,14 +46,55 @@ npm create vite@latest starter-app -- --template react
...
+ENABLE_TESTNETS=true
+```
+
+Note that we'll use `import.meta.env` to access these variables (below) in the `wagmi.js` file. Depending on your frontend setup, it may differ. With Vite, you use `import.meta.env` and prefix the variable with `VITE_`. But, if you're using Next.js, you'd use `process.env` and prefix with `NEXT_PUBLIC_`.
+
+<Tabs groupId="sdk">
+<TabItem value="jsv2" label="wagmi v2">
+
+```jsx title="src/wagmi.js"
+import "@rainbow-me/rainbowkit/styles.css";
```
Comment (dtbuchholz) [inline docs/playbooks/frameworks/wagmi.md:62]: idk if helpful, but if the Studio ever needs to adopt wagmi v2, this should explain the bulk of it. i just didnt' pass custom RPC providers and opted for wallet-only RPCs.

File: src/components/projects.ts [hunk 2]
```diff
@@ -28,6 +28,16 @@ export default [
     github: "https://github.com/WeatherXM",
     twitter: "https://twitter.com/WeatherXM",
     details: `WeatherXM is a network that's powered by community owned devices, purpose built by WeatherXM for weather data collection. As part of the network, devices are incentivized to share data, which is sent to various stations throughout the globe.\n\nOne of the challenges with this approach is not only ensuring that the raw device data is available when it's needed, but as a decentralized network, its infrastructure must also make sure that data is stored in a decentralized fashion instead of centralized infrastructure.\n\nAs devices send data to the network, it's replicated to a hot for immediate and TTL retrieval, which is pertinent for compute over that data. And for persistence guarantees, this data is also replicated to Filecoin for long-term storage guarantees, which can help the network ensure that data is available for years to come.`,
+    blogs: [
```
Comment (dtbuchholz) [inline src/components/projects.ts:31]: small optional feature to show links to blog posts about partners, if available (only wxm has anything)

File: docs/playbooks/integrations/lit.md [hunk 3]
```diff
@@ -0,0 +1,730 @@
+---
```
Comment (dtbuchholz) [inline docs/playbooks/integrations/lit.md:1]: this is the bulk of the new stuff. Lit has a bunch of features, and I'm not quite sure how exactly it works. but, if you try to read data from [lit_encrypt_11155111_1314](https://testnets.tableland.network/api/v1/query?statement=SELECT%20*%20from%20lit_encrypt_11155111_1314), you can see what the raw/encrypted data looks like. 

you basically set up an access control condition, which can be _any_ arbitrary code (e.g., JS script). they have built-in methods for reading EVM contracts, which is what i do below. i.e., my access control calls the Tableland registry contract's `ownerOf` method and requires the one trying to decrypt is the owner of the table. this condition is used when writing data to the table as well as when decrypting table data.

File: docs/playbooks/integrations/lit.md [hunk 4]
```diff
@@ -0,0 +1,730 @@
...
+
+## Background
+
+[Lit Protocol](https://www.litprotocol.com/) is a key management network for decentralized signing and encryption. The premise is that instead of a single entity holding the entire private key, each node within the network holds a unique private key share. In order to obtain the final signature or decrypted content, a pre-defined threshold for the number of nodes is required for combining signatures or decryption shares. In other words, if your threshold was 2/3 of 100 nodes, you'd need to gather decryption or signature shares from at least 67 nodes.
+
+Naturally, this is a perfect fit for encrypting and decrypting Tableland data! We'll walk through a simple example of how to use Lit to encrypt data before writing to Tableland, and then decrypting that data upon table reads. Note that Lit has its own built-in access control system that could be used in tandem with Tableland's access control, too.
+
+## Installation
+
+We'll first install the Tableland SDK and Lit SDK (version 3.x). This example walks uses NodeJS, and you'll also need the `siwe` package to generate a Lit `AuthSig`, described below.
```
Comment (joewagner) [inline docs/playbooks/integrations/lit.md:24]: Is "This example walks uses NodeJS," a typo?

**Review by joewagner — COMMENTED** [review]
@dtbuchholz this is great!  I didn't actually run the Lit example, but it's really interesting.  I'll try to build soemthing with it once these docs are live.

I think I found one typo, left an inline comment, otherwise lgtm.

**Comment by dtbuchholz** [comment]
@joewagner nice typo catch, fixed! and i _just_ squeezed in a GH action to fix an issue where Algolia search was not indexing new docs pages; it calls the Algolia API to trigger reindexing upon merges to main

**Review by joewagner — APPROVED** [review]

**Pull request merged**